holmscan/cli.py

```diff
@@ -152,6 +155,21 @@
 """
 
 
+def _print_format(data, headers):
+    if print_format == constants.OUTPUT_FORMAT_TABLE:
+        print(tabulate(data, headers, **tabulate_args))
+    elif print_format == constants.OUTPUT_FORMAT_YAML:
+        data_with_headers = []
+        for scan in data:
+            scan_with_headers = {}
+            for header, entry in zip(headers, scan):
+                scan_with_headers[header.lower()] = entry
+            data_with_headers.append(scan_with_headers)
+        print(dump(data_with_headers, **yaml_dump_args))
+    else:
+        raise HolmscanConfigException("unsupported print format")
```

Comment:
Agreed, changed the code. Just for clarity, this else-clause should never be hit because validation of the configuration is done much earlier in the code.